vnext/Folly/Folly.vcxproj

```diff
@@ -311,6 +340,16 @@
                 }
             },
             "DevelopmentDependency": false
+        },
```

Comment:
You added this, but it looks like the manifest wasn't re-generated?

Reply:
fixed! didn't realize you had to clean + rebuild to generate